tests/components/sun/test_trigger.py

```diff
@@ -775,17 +775,17 @@ async def test_if_action_before_sunset_no_offset_kotzebue(hass, calls):
         },
     )
 
-    # sunrise: 2015-07-24 07:17:24 local, sunset: 2015-07-25 03:16:27 local
-    # sunrise: 2015-07-24 15:17:24 UTC,   sunset: 2015-07-25 11:16:27 UTC
-    # now = sunrise + 1s -> 'before sunrise' not true
-    now = datetime(2015, 7, 25, 11, 16, 28, tzinfo=dt_util.UTC)
+    # sunrise: 2015-07-24 07:21:12 local, sunset: 2015-07-25 03:13:33 local
+    # sunrise: 2015-07-24 15:21:12 UTC,   sunset: 2015-07-25 11:13:33 UTC
+    # now = sunset + 1s -> 'before sunset' not true
```

Comment:
Why does this change?

Reply:
The new Astral both produces different times and provides times down to the MS level. To deal with that the tests need to be re-written. 
The times need to be changed for new Astral's more accurate times, and to deal with MS rounding, I had to add +1s or -1s to tests that would fail from not passing the MS marker.
I got confirmation here:
https://github.com/home-assistant/core/pull/48282#issuecomment-808855672